lwb/metalang/cfg/cfg/src/main/java/mb/cfg/convert/Parts.java

```diff
@@ -172,6 +172,14 @@ void forOneSubtermAsTypeInfo(String name, Consumer<TypeInfo> consumer) {
         getOneSubtermAsTypeInfo(name).ifSome(consumer);
     }
 
+    Option<String> getOneSubtermAsConstructorName(String name) {
+        return getOneSubterm(name).map(IStrategoAppl.class::cast).map(IStrategoAppl::getName);
```

Comment:
It would be nicer if this threw a `InvalidAstShapeException` explaining what is expected (i.e., `throw new InvalidAstShapeException("term application", term)`) instead of this failing with `ClassCastException` when the subterm is not a `IStrategoAppl`.